javascript/javascript1/week1/Future_Age.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html lang="en">
+<head>
+    <title>A future age calculator</title>
+    <meta name="viewport" charset="utf-8" width="device-width, initial-scale=1.0">
+
+</head>
+
+<body>
+
+
+    <script>
+        const yearOfBirth = 1984;
```

Comment:
add a script to file

Reply:
Hi, thanks a lot. Do you mean a separate js file?

Reply:
yes